### ISSUE TITLE
Traduction missing on the dashboard

### ISSUE THREAD

**Issue by Pikabois** [KB-RolePlay/tools-issues#119]
### Version

v2.3.36

### Description

the "add-note-button" translation in french is missing

### Reproduction steps

```bash
1. Go to the dasboard
2. Find someone and try to add a note to his profile
```

### Screenshots

![image](https://github.com/KB-RolePlay/tools-issues/assets/46120142/52c2fe8a-c089-4dff-8210-597c4ce4d492)

**Comment by Pikabois** [comment]
Same bug for the settings modifications update button (https://dashboard.kobralost-rp.com/admin/1/settings)
![image](https://github.com/KB-RolePlay/tools-issues/assets/46120142/ed0b61c0-ee74-4fb0-b177-a293935a7760)

**Comment by hugoheml** [comment]
Resolved on `v2.4.30`, thanks for your report!
